test/tests/models/Message.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Message = require('../../../models/Message');
+
+describe('Message model', () => {
+  describe('creating Message', () => {
+    let message;
+
+    beforeEach(() => {
+      message = new Message({ from_email: 'ssing128@gmail.com', to_email:'ssing128@gmail', body: 'i love you' });
+    })
+
+    it('should have from email', () => {
+      message.should.have.property('from_email', message.from_email);
+    }) 
+
+    it('should have to email', () => {
+      message.should.have.property('to_email', message.to_email);
+    }) 
+
+    it('should have body', () => {
+      message.should.have.property('body');
```

Comment:
Why is this test different than the ones just above it?

Reply:
I have removed this file from test because i will rewrite the tests.